bitnami/discourse/templates/_helpers.tpl

```diff
@@ -120,7 +120,29 @@ imagePullSecrets:
 {{- end -}}
 
 {{/*
-Get the user defined LoadBalancerIP for this release.
+Return the Discourse secret name
+*/}}
+{{- define "discourse.discourseSecretName" -}}
```

Comment:
I think it'd be nicer to call this simply `discourse.secretName`. As this chart has multiple secrets coming from different sub-charts, you can take a look at [bitnami/airflow](https://github.com/bitnami/charts/blob/master/bitnami/airflow/templates/_helpers.tpl)  which follows this approach.

https://github.com/bitnami/charts/blob/37e4526071788e0adb2c14d9b427ecf42cc6c64d/bitnami/airflow/templates/_helpers.tpl#L328
https://github.com/bitnami/charts/blob/37e4526071788e0adb2c14d9b427ecf42cc6c64d/bitnami/airflow/templates/_helpers.tpl#L311
https://github.com/bitnami/charts/blob/37e4526071788e0adb2c14d9b427ecf42cc6c64d/bitnami/airflow/templates/_helpers.tpl#L290